pathfinder/tests/fixtures/core.py

```diff
@@ -1,42 +1,47 @@
 import random
 from typing import List
 from unittest.mock import Mock
-
 import pytest
-from eth_utils import to_checksum_address, encode_hex, keccak
+from coincurve import PrivateKey
+from eth_utils import to_checksum_address, remove_0x_prefix
 from raiden_libs.contracts import ContractManager
+from raiden_libs.utils import public_key_to_address
 from web3 import Web3
 from web3.contract import Contract
-
 from pathfinder.pathfinding_service import PathfindingService
+from pathfinder.tests.config import NUMBER_OF_CHANNELS
 from pathfinder.token_network import TokenNetwork
+from pathfinder.utils.types import Address
 
 
 @pytest.fixture
 def populate_token_networks(
         token_networks: List[TokenNetwork],
         token_network_contracts: List[Contract],
+        addresses: List[Address],
+        private_key_seeds: List[str],
 ) -> None:
-
+    random.seed(NUMBER_OF_CHANNELS)
```

Comment:
Explicit seed is good for reproducible testing but why `NUMBER_OF_CHANNELS`? `0` would be just fine too.

Reply:
to make it look cool!

Reply:
It's confusing. People reading this will wonder why the seed is exactly `NUMBER_OF_CHANNELS` so please change that or document that you just use that variable to make it look cool according to your standards of cool.

Reply:
Added a comment.

Reply:
don't use fixed seed for regular testing - running the test every time with set of variables derived from different seed values reveals more potential problems.

Reply:
@pcppcp These `TokenNetwork`s are intended for unit testing. We need those networks to be deterministic for reproducibility. Some networks might be partitioned in which case tests would probably fail, so it's good to have one working network to run tests on.

Alternatively we could use a real random seed and log it each time for reproducibility, so we can then debug against a fixed seed. But still, having tests randomly fail and pass doesn't seem like a good idea to me.

Reply:
Have you considered generating the network with `networkx` library? It can create various topologies that are guaranteed to be connected. Or re-generate network until you get connected graph.
Logging the seed is a good idea. You could set envvar/option that's set to i.e. `date +%s` from Travis or use python's `time` as a fallback.